### ISSUE TITLE
Mod parsing - Small bug regarding a helmet enchant

### ISSUE THREAD

**Issue by drevtpohlman** [maxensas/xiletrade#26]
The "Spark fires 3 additional projectiles" enchant is being ignored while price-checking, while other enchants seem to work just fine.

I think the issue here is the difference in the wording used on the official trade site to search for this enchant, which is "spark fires an additional projectile" with the minimum value set to 3, rather than "spark fires # additional projectiles" like what some might expect.

There could be more discrepancies in search terms on the official site causing enchants/mods to be omitted, but this is just one that I noticed.

Screen of spark enchant and its overlay:
![image](https://github.com/maxensas/xiletrade/assets/51380416/579221c2-85f8-4047-aab6-a360b2c4d861)

Screen of another random enchant and its overlay:
![image](https://github.com/maxensas/xiletrade/assets/51380416/e9fd635f-36c3-49cf-b719-29dc26e5acea)

I've also added the Ctrl+C data of the helm with the spark enchant if it helps:

```
Item Class: Helmets
Rarity: Magic
Virile Deicide Mask of the Starfish
--------
Quality: +20% (augmented)
Evasion Rating: 262 (augmented)
Energy Shield: 65 (augmented)
--------
Requirements:
Level: 67
Dex: 73
Int: 88
--------
Sockets: B 
--------
Item Level: 92
--------
Spark fires 3 additional Projectiles (enchant)
--------
+73 to maximum Life
Regenerate 17.3 Life per second

```

**Comment by maxensas** [comment]
Hi, thanks for your report. It's appreciated.

We already identified this issue exactly like you describe. 
It exist since 2/3 leagues i would say when GGG did an update to their filters. 
I already started to work on a new algorythm to identify thoses mods dynamically in all languages, not just english instead of parsing them one by one (see ParsingRules.json file) and inject the numeric value in input that we do not have on the mod recognized by the trade website.
